edbterraform/data/terraform/aws/modules/specification/files.tf

```diff
@@ -2,7 +2,7 @@
 # If ssh_key.public_path and ssh_key.private_path are defined,
 # overwrite the default keys.
 locals {
-  ssh_user_count = var.spec.ssh_user != null ? 1 : 0
+  ssh_user_count = var.spec.images != null ? 1 : 0
```

Comment:
For azure, you also check if `var.spec.kubernetes != null`:
```
ssh_user_count = var.spec.kubernetes != null || var.spec.images != null ? 1 : 0
```

This check is not needed here?

Reply:
@jt-edb
Currently, only azure's kubetenete's needs an ssh key since AWS/GCloud kubernetes do not make use of it with our current integration.

Would it be better to remove this check and create the ssh keys even if not used by the resources? So the only check would be whether to override the default ssh key or not.